src/vite-bundle/src/Twig/TypeExtension.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Pentatrion\ViteBundle\Twig;
+
+use Pentatrion\ViteBundle\Service\Debug;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+final class TypeExtension extends AbstractExtension
+{
+    public function getFilters(): array
+    {
+        return ['stringify' => new TwigFilter('stringify', [Debug::class, 'stringify'], ['is_safe' => ['html']])];
```

Comment:
`stringify` is quite a generic name and I think we run the risk of conflicting with another filter here?

I think we should prefix  the filter name with the bundle name, something like `symfonyvite_stringify` maybe?

The array key is ignored too, so this could be changed to:

```suggestion
        return [new TwigFilter('symfonyvite_stringify', [Debug::class, 'stringify'], ['is_safe' => ['html']])];
```

Reply:
Normally only the `ProfilerController` add this extension, but you're right, I've changed.